src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -392,14 +390,6 @@ static void _applyProperty(SvgLoaderData& loaderData, SvgNode* node, Shape* vg,
 }
 
 
-static unique_ptr<Shape> _shapeBuildHelper(SvgLoaderData& loaderData, SvgNode* node, const Box& vBox, const string& svgPath)
-{
-    auto shape = Shape::gen();
-    if (_appendShape(loaderData, node, shape.get(), vBox, svgPath)) return shape;
-    else return nullptr;
-}
```

Comment:
shapeAppend() -> shapeBuildHelper() doesn't have any special functions. That's why this refactoring looks efficient, but xxxBuildHelper() and append*Shape() have different internal structures. 

The reason why the append prefix was used is probably because shape->append*() APIs were used with _recognizeShape() (before the [Extract-Method](https://refactoring.guru/ko/extract-method) was applied).

Since append*Shape() became meaningless, I think _appendClipShape(), _appendClipUseNode() should also be _clipShapeBuildHelper(), _clipUseNodeBuildHelper().